pkg/nodeset/v1alpha1/types.go

```diff
@@ -28,9 +28,16 @@ const (
 	NodeClassResourceFile      NodeClassResourceType = "File"
 	NodeClassResourceReference NodeClassResourceType = "Reference"
 
-	// Fixed namespace for now to satisfy TPR
-	// TODO: Get rid of namespace when we port to TPR successor
-	TPRNamespace = "default"
+	NodeClassContentAnnotationKey     = "nodeset.k8s.io/nodeclass-content"
+	NodeClassContentHashAnnotationKey = "nodeset.k8s.io/nodeclass-content-hash"
+
+	NodeSetGenerationAnnotationKey = "nodeset.k8s.io/nodeset-generation"
+
+	NodeSetNameLabelKey = "nodeset.k8s.io/controller"
+	ControllerLabelKey  = "node.k8s.io/controller"
```

Comment:
Should we use node.k8s.io everywhere or split annotations into node.k8s.io and nodeset.k8s.io ?